lib/widgets/screens/inbox/inbox_invites_sent.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:mm_flutter_app/constants/app_constants.dart';
+import 'package:mm_flutter_app/widgets/atoms/inbox_list_tile.dart';
+
+class InboxInvitesSentScreen extends StatelessWidget {
+  const InboxInvitesSentScreen({super.key});
+
+  InboxListTile _createTestTile(
+    BuildContext context,
+    AppLocalizations l10n,
+    int tileIndex,
+  ) {
+    // TODO(m-rosario): Replace mock data with backend data.
+    final DateTime date = DateTime.now()
+        .subtract(Duration(days: tileIndex * pow(1.4, tileIndex).floor()));
```

Comment:
Nothing blocking here, but wanted to understand this formula. Why did you choose to raise 1.4 to the power of tileIndex? Thanks!